sklearn/gaussian_process/correlation_models.py

```diff
@@ -17,9 +17,9 @@ def absolute_exponential(theta, d):
     Absolute exponential autocorrelation model.
     (Ornstein-Uhlenbeck stochastic process)::
 
-                                            n
-        theta, dx --> r(theta, dx) = exp(  sum  - theta_i * |dx_i| )
-                                          i = 1
+                                          n
+        theta, d --> r(theta, d) = exp(  sum  - theta_i * |d_i| )
```

Comment:
Oh... okay.. I had assumed that it was named dx earlier and was renamed to d following a deprecation cycle...
Anyway just now referred a few CRAN packages / Matlab packages all follow the same convention of `d` only... So I guess this is fine :)